### DISCUSSION TITLE
add scoop in downloads section of github and possibly remove portableapps

### DISCUSSION THREAD

**Discussion by t3dium** [FreeTubeApp/FreeTube#2539] [General]
As its a better way of installing portable apps and is already listed on scoop's main repos: `scoop install freetube`

> While they should be safe, download at your own risk

it should also be 100% safe as it simply fetches the download from your github repository which can be verified using `scoop cat`

![image](https://user-images.githubusercontent.com/83690012/188330359-3a5941b8-bf82-40b6-8eab-b98ef8fb6005.png)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Opened https://github.com/FreeTubeApp/FreeTube/pull/2553. Portableapps will not be removed because we want to give the users of FT all the options there are.
